packages/fxa-auth-server/lib/routes/account.ts

```diff
@@ -1718,6 +1801,19 @@ export const accountRoutes = (
       },
       handler: (request: AuthRequest) => accountHandler.login(request),
     },
+    {
+      method: 'POST',
+      path: '/account/login/third_party',
+      options: {
+        validate: {
+          payload: {
+            token: isA.string().required(),
```

Comment:
Are we planning on using this endpoint for multiple IDPs? I assume so from the name. It might be worth considering what the payload might look like once we add more. Adding a `provider` field (or something) now will make the logic easier later when we go from 1 to 2+.

Reply:
@dannycoates correct, I was hoping to have just one endpoint that we call and setup new users etc. I'll add the `provider` field 👍🏽

Reply:
cool, i was thinking `provider` wouldn't be optional and we'd send 'google' from the start. Then we don't have to worry about the (small) transition time where no provider implicitly means google.